# HG changeset patch
# User jwe
# Date 749184798 0
#      Tue Sep 28 02:53:18 1993 +0000
# Node ID c8f575fdcc91874ab1b7c6a492acabb90b15c81b
# Parent  ef32610d62cdb94e54cb31fb1ff5922cdebe5769
[project @ 1993-09-28 02:53:18 by jwe]
Don't say #pragma interface since there's no corresponding
implementation.

diff --git a/liboctave/NLP.h b/liboctave/NLP.h
--- a/liboctave/NLP.h
+++ b/liboctave/NLP.h
@@ -19,20 +19,16 @@ You should have received a copy of the G
 along with Octave; see the file COPYING.  If not, write to the Free
 Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
 
 */
 
 #if !defined (_NLP_h)
 #define _NLP_h 1
 
-#ifdef __GNUG__
-#pragma interface
-#endif
-
 #include "Objective.h"
 #include "Bounds.h"
 #include "LinConst.h"
 #include "NLConst.h"
 #include "Matrix.h"
 
 #ifndef Vector
 #define Vector ColumnVector
